doc/UserGuide.md

```diff
@@ -111,6 +111,10 @@ Examples:
   `viewall 1`<br>
   Views all details of the 1st person in the results of the `find` command.
 
+### Sorting address book by name before displaying : `sortAndList`
+Sorts all entries by name in alphabetical order before displaying.
+Format: sortAndList
```

Comment:
seems a bit long for a command key. 
You can actually just make it name `sort` and let the listing part be its natural behaviour